dataeng/jobs/analytics/RetirementJobEdxTriggers.groovy

```diff
@@ -0,0 +1,145 @@
+/* UserRetirementJobEdxTriggers.groovy
+ *
+ * Defines jobs which trigger downstream user retirement jobs for each edX
+ * environment.
+ */
+
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.jenkins_build_team_security
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+
+class RetirementJobEdxTriggers{
+    public static def job = { dslFactory, allVars ->
+        List jobConfigs = [
```

Comment:
Our current established pattern is to define job configs in analytics-secure under the job-configs directory---I think this case is no different than other jobs.  I also noticed that our authorization/permissions for these jobs were defined inside the job DSL, but I think we should also move those to the yaml job configs as is done in our other jobs.

Reply:
Yes sure, I have done that change